core/app/[locale]/(default)/account/(tabs)/orders/_components/order-details.tsx

```diff
@@ -0,0 +1,301 @@
+import { removeEdgesAndNodes } from '@bigcommerce/catalyst-client';
+import { getFormatter, getLocale, getTranslations } from 'next-intl/server';
+
+import { Link } from '~/components/link';
+import { Button } from '~/components/ui/button';
+import { cn } from '~/lib/utils';
+
+import { OrderDetailsDataType } from '../page-data';
+
+import { assembleProductData, ProductSnippet } from './product-snippet';
+
+const OrderState = async ({ orderState }: { orderState: OrderDetailsDataType['orderState'] }) => {
+  const locale = await getLocale();
+  const t = await getTranslations({ locale, namespace: 'Account.Orders' });
+  const format = await getFormatter();
+  const { orderId, orderDate, status } = orderState;
+
+  return (
+    <div className="mb-6 flex flex-col gap-6 md:flex-row">
+      <div>
```

Comment:
can we remove this `div`?

Reply:
I wrap order number & date in separate flex item via div.